.github/workflows/common-app-build.yml

```diff
@@ -61,6 +61,10 @@ jobs:
         id: branch-name
         uses: tj-actions/branch-names@v5.1
 
+      # Declares the repository safe and not under dubious ownership.
+      - name: Add repository to git safe directories
+        run: git config --global --add safe.directory $GITHUB_WORKSPACE
```

Comment:
toto vies spravit v actions/checkout@v3 podla ich dokumentacie https://github.com/actions/checkout, v readme je to v usage pod tymto:

```
    # Add repository path as safe.directory for Git global config by running `git
    # config --global --add safe.directory <path>`
    # Default: true
    set-safe-directory: ''
```

Reply:
ten path bohužiaľ tiež nepomohol

Reply:
tak to je nemile